sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -406,6 +406,91 @@ Follow the guide to [add app roles in your application and assign to users or gr
         return "Admin message";
     }
     ```
+#### Support Conditional Access in web application.
+  
+This starter supports [Conditional Access] policy. By using [Conditional Access] policies, you can apply the right **access controls** when needed to keep your organization secure. **Access controls** has many concepts, [Block Access] and [Grant Access] are important. In some scenarios, this stater will help you complete [Grant Access] controls. 
+ 
+In [Resource server visiting other resource server] scenario(For better description, we think that resource server as **webapiA** and the other resource servers as **webapiB**), When we configure the webapiB application with Conditional Access(such as [multi-factor authentication]), this stater will help us send the Conditional Access information of the webapiA to the web application and the web application will help us complete the Conditional Access Policy. As shown below:
+
+  ![aad-conditional-access-flow.png](resource/aad-conditional-access-flow.png)
```

Comment:
In this image
1. Step 1: `and acquire token for Web API A` -> `and got token for Web API A`.
2. Step 4: `The claims field in error is important` -> `Error information is stored in claims field`.

Reply:
Thanks, got it.